src/gov-action-contracts/AIPs/AIP1Point2Action.sol

```diff
@@ -41,9 +41,11 @@ contract AIP1Point2Action {
         );
     }
 
-    function setProposalThreshold(IL2ArbitrumGoverner gov, uint256 newProposalThreshold) internal {
+    function setProposalThreshold(IL2ArbitrumGoverner gov, uint256 _newProposalThreshold)
+        internal
+    {
         bytes memory setProposalThresholdCalldata = abi.encodeWithSelector(
-            IL2ArbitrumGoverner.setProposalThreshold.selector, newProposalThreshold
+            IL2ArbitrumGoverner.setProposalThreshold.selector, _newProposalThreshold
```

Comment:
this shadowed the global constant I know this is passed already but can't help myself to resolve the compiler warning